quiz.html

```diff
@@ -3,53 +3,121 @@
 	<head>
 		<title>Skin type Quiz</title>
 		<link rel="stylesheet" href="style.css"/>
+        <script type="text/javascript" src="quiz.js"></script>

```

Comment:
In my final layout, I began creating the skin type quiz using javascript and linked the file to the quiz page.